### ISSUE TITLE
Python error on style load:  unsupported operand type(s) for +: 'dict' and 'str'

### ISSUE THREAD

**Issue by frodrigo** [maptiler/qgis-maptiler-plugin#120]
```
TypeError: unsupported operand type(s) for +: 'dict' and 'str' 
Traceback (most recent call last):
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/browser_mapitem.py", line 46, in handleDoubleClick
    self._add_custom_to_canvas()
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/browser_mapitem.py", line 338, in _add_custom_to_canvas
    self._add_vector_to_canvas(data_key='custom')
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/browser_mapitem.py", line 194, in _add_vector_to_canvas
    self._add_vtlayer_from_style_json(
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/browser_mapitem.py", line 222, in _add_vtlayer_from_style_json
    renderer, labeling = converter.get_renderer_labeling(
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/gl2qgis/converter.py", line 102, in get_renderer_labeling
    renderer, labeling = parse_layers(source_layers, style_json_data.get('id'))
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/gl2qgis/gl2qgis.py", line 821, in parse_layers
    st = parse_fill_layer(json_layer, style_name)
  File "/home/fred/.local/share/QGIS/QGIS3/profiles/default/python/plugins/qgis-maptiler-plugin/gl2qgis/gl2qgis.py", line 503, in parse_fill_layer
    raster_fill_symbol = QgsRasterFillSymbolLayer(os.path.join(SPRITES_PATH, fill_pattern + ".png"))
TypeError: unsupported operand type(s) for +: 'dict' and 'str'


Version de Python : 3.8.3 (default, May 17 2020, 18:15:42) [GCC 10.1.0] 
Version de QGIS : 3.14.0-Pi Pi, exported 
```

The style from here: https://vecto-dev.teritorio.xyz/styles/teritorio-tourism-dev/style.json

**Comment by lazaa32** [comment]
Plugin does not support fill pattern change over zoom levels.
I am not even sure if that is something we can implement.
@Kanahiro could you have a look at this and make a research if it is possible in QGIS, please?
```
fill_pattern = {dict} {'stops': [[11, 'wetland8'], [12, 'wetland16'], [13, 'wetland32'], [14, 'wetland64']]}
```

**Comment by lazaa32** [comment]
Hi @frodrigo,
@Kanahiro fix is released in v1.1.2 which is awaiting approval now (usually takes one day). Could you please upgrade version of the plugin to v1.1.2 (probably available tomorrow) and confirm it works for you? Thanks.

**Comment by frodrigo** [comment]
Now it loads the style. Thank you a lot for all this fix.
